modules/raft/src/main/java/org/apache/ignite/raft/jraft/rpc/impl/IgniteRpcClient.java

```diff
@@ -87,23 +87,22 @@ public ClusterService clusterService() {
     ) {
         CompletableFuture<Message> fut = new CompletableFuture<>();
 
-        fut.orTimeout(timeoutMs, TimeUnit.MILLISECONDS).
-            whenComplete((res, err) -> {
-                assert !(res == null && err == null) : res + " " + err;
+        fut.whenComplete((res, err) -> {
```

Comment:
This change seems to change the semantics. Before the change, `whenComplete()` callback would see the `TimeoutException`, but now it won't.

Reply:
We still set the timeout on sending the operation.
It just has excessive usage of `orTimeoput`, that I removbed.